org.mixedrealitytoolkit.input/Utilities/FollowJoint.cs

```diff
@@ -36,22 +36,10 @@ internal class FollowJoint : MonoBehaviour, ISerializationCallbackReceiver
         [HideInInspector]
         private Handedness hand;
 
-        /// <summary>
-        /// The hand on which to track the joint.
-        /// </summary>
-        [Obsolete("Please change the Hand value on the jointPoseSource instead")]
-        protected Handedness Hand { get => JointPoseSource.Hand; set => JointPoseSource.Hand = value; }
-
         [SerializeField]
         [HideInInspector]
         private TrackedHandJoint joint;
```

Comment:
Are these hidden serialized fields still used, or should they be removed too, since we're removing their properties?

Reply:
Correct, I missed that.  Thank you for catching that and letting me know.  I removed them in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/774/commits/39287ee594ca665e3cca42376ff0802d3f9d5064 , resolving.